tests/snaps/vine/option_party/stats.txt

```diff
@@ -1,15 +1,15 @@
 
 Interactions
-  Total                  9_682
-  Annihilate             5_230
+  Total                  9_763
+  Annihilate             5_273
```

Comment:
Was this change assessed?

Reply:
Yes; it increased in https://github.com/VineLang/vine/pull/45/commits/16096d4275c0d4c5f382f24c2aca6258f95ae230 because I changed one of the string literals it prints, and made it longer. https://github.com/VineLang/vine/pull/45/commits/e4eee5db5e8ea4875468da18f6bc42d3f93ef160 changed the file from using `is_none_or` to `=>`, without changing the behavior at all, and it had a negative delta.